qa/common/infer_util.py

```diff
@@ -55,15 +57,21 @@ def infer_exact(tester, pf, tensor_shape, batch_size,
                 model_version=None, swap=False,
                 outputs=("OUTPUT0", "OUTPUT1"), use_http=True, use_grpc=True,
                 skip_request_id_check=False, use_streaming=True,
-                correlation_id=0):
+                correlation_id=0, use_shared_memory=False):
     tester.assertTrue(use_http or use_grpc or use_streaming)
     configs = []
     if use_http:
-        configs.append(("localhost:8000", ProtocolType.HTTP, False))
+        if use_shared_memory:
+            configs.append(("localhost:8000", ProtocolType.HTTP, False, True))
+        configs.append(("localhost:8000", ProtocolType.HTTP, False, False))
```

Comment:
So if `use_shared_memory == True`, the "no shared memory" case will still be tested?

Reply:
include_shared_memory_test might be a better name to imply that in addition to regular testing also want to do shared-memory testing.

Reply:
Done